patch_check.go

```diff
@@ -43,7 +43,6 @@ func patchCheck(image string, ctx *cli.Context) {
 		// then we do nothing.
 		de := err.(dockerError)
 		if de.exitCode == 100 || de.exitCode == 101 {
-			exitWithCode(int(de.exitCode))
```

Comment:
Removed this line, because we don't want the program to exit at this point.
Also the commend suggest to do nothing so we will just continue to run 
normally at this point.

Reply:
This seems to fix a bug and hence should be moved into a dedicated commit.

Reply:
Better into a separate PR where the issue can be discussed separately (to avoid cross postings).

Reply:
I had a look at the specific path and I wouldn't remove the line here. The idea is that `zypper-docker` returns `zypper`'s error codes. If we remove the line, we wouldn't do that in this specific case.

Reply:
@vrothberg: I partly agree. The only problem is, that if I do not remove it the pchkc command does not terminate properly which causes the committed image to not be deleted afterwards. 
So I decided to put it in this PR, because it is related to the changes made. 
If you disagree, I will put it in a separate PR.

Reply:
If you want to have it in this PR, then we should move it into a 2nd commit on top of the current one. But I am okay with doing that in a separate PR as well.

The problem with removing this line still holds. `zypper-docker` must return `zypper` exit codes, otherwise, the bug fix is causing another bug.